rmf_robot_sim_common/include/rmf_robot_sim_common/slotcar_common.hpp

```diff
@@ -317,7 +310,10 @@ class SlotcarCommon
 
   void path_request_cb(const rmf_fleet_msgs::msg::PathRequest::SharedPtr msg);
 
-  void ackmann_path_request_cb(
+  void diff_drive_path_request_cb(
```

Comment:
Nit: since we are not exactly using these functions as a callback, we should call it something like `handle_diff_drive_path_request` or something along those lines.